spec/directions/bake_chapter_summary_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeChapterSummary do
+
+  let(:chapter) do
+    chapter_element(
+      <<~HTML
+        <div data-type='page' id ="01">
+          <h1 data-type="document-title" itemprop="name">First Title</h1>
```

Comment:
Add an ID on this title to see that it gets populated in the `<a href="#that_id">` tag.  Ditto for the second page.